addons/account/controllers/tests_shared_js_python.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from odoo import http
+from odoo.http import request
+
+
+class TestsSharedJsPython(http.Controller):
+
+    @http.route('/account/init_tests_shared_js_python', type='http', auth='user', website=True)
```

Comment:
@william-andre If I don't specify this key, since it's a frontend page, it failed when `website` is installed. This is because `website` inherits the frontend_layout and absolutely want some values that has to be there. If I removed this key:
```
Traceback (most recent call last):
  File "<187>", line 1579, in template_187
  File "<187>", line 1210, in template_187_content
  File "/home/odoo/src/odoo/odoo/addons/base/models/ir_qweb.py", line 2510, in _load_values
    return get_value()
  File "<187>", line 1200, in template_187_t_cache_0_cache
  File "<187>", line 131, in template_187_t_cache_0
KeyError: 'website'
```

Reply:
@william-andre To be able to use `t-owl-component` :(